src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/Dataflow/FlowAnnotations.cs

```diff
@@ -700,9 +700,9 @@ internal void ValidateMethodAnnotationsAreSame(MethodDesc method, MethodDesc bas
             if (methodAnnotations.ParameterAnnotations != null || baseMethodAnnotations.ParameterAnnotations != null)
             {
                 if (methodAnnotations.ParameterAnnotations == null)
-                    ValidateMethodParametersHaveNoAnnotations(baseMethodAnnotations.ParameterAnnotations!, method, baseMethod, method);
+                    ValidateMethodParametersHaveNoAnnotations(baseMethodAnnotations.ParameterAnnotations!, baseMethod, method);
                 else if (baseMethodAnnotations.ParameterAnnotations == null)
-                    ValidateMethodParametersHaveNoAnnotations(methodAnnotations.ParameterAnnotations, method, baseMethod, method);
```

Comment:
I would prefer we also leave all the files in the DataFlow directory untouched (just suppress - maybe via editor config).
Lot of the code here is sort of shared with the linker (we compare the source files between the two repos manually).
I would rather we tackle this cleanup once we have linker in the same repo and can implement even deeper sharing.

Reply:
this file is aot specific and local so I kept it but can revert it if you want